StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1133CodeFixProvider.cs

```diff
@@ -77,7 +77,9 @@ private static List<AttributeListSyntax> GetNewAttributeList(AttributeListSyntax
 
             for (var i = 0; i < attributeList.Attributes.Count; i++)
             {
-                var newAttributes = SyntaxFactory.SingletonSeparatedList(attributeList.Attributes[i]);
+                var newAttributes = SyntaxFactory.SingletonSeparatedList(
+                    attributeList.Attributes[i].WithLeadingTrivia(
+                        attributeList.Attributes[i].GetLeadingTrivia().WithoutLeadingWhitespace()));
```

Comment:
I don't think there is another way. There is of course the possibility to add a new extension method to `TriviaHelper`.
Something with a syntax like this:
```csharp
internal static TSyntax WithoutLeadingWhitespace<TSyntax>(this TSyntax node, bool endOfLineIsWhitespace = true) where TSyntax : SyntaxNode
{
    ...
}
```